worlds/shorthike/Rules.py

```diff
@@ -11,8 +11,20 @@ def create_rules(self, location_table):
             forbid_items_for_player(multiworld.get_location(loc["name"], player), self.item_name_groups['Maps'], player)
             add_rule(multiworld.get_location(loc["name"], player),
                 lambda state: state.has("Shovel", player))
+
+        # Shop Rules
         if loc["purchase"] and not options.coins_in_shops:
             forbid_items_for_player(multiworld.get_location(loc["name"], player), self.item_name_groups['Coins'], player)
+        if loc["purchase"] >= get_min_shop_logic_cost(self) and options.shop_check_logic != 0:
```

Comment:
You may want to do this so that the cost multiplier is accounted for in logic instead of the original prices
```suggestion
        if loc["purchase"] * options.cost_multiplier / 100 >= get_min_shop_logic_cost(self) and options.shop_check_logic != 0:
```

Reply:
No, since the min shop logic option doesn't scale with the cost multiplier this wouldn't be desirable. I will however note in the description for the option that it is based on vanilla prices.

Reply:
This makes it scale, I'm confused what you mean?

Reply:
This makes it scale, I'm confused what you mean?

Reply:
The choices for the Min Shop Logic Application option are only 40, 100 and 400 (the only 3 vanilla shop prices). These choices don't change with the cost multiplier and would only make it confusing to apply the multiplier to the shop costs when calculating the logic if you're only able to choose between these three set prices.